### DISCUSSION TITLE
firstbot

### DISCUSSION THREAD

**Discussion by giscus** [brainboxdotcc/dpp-comments#11] [General]
# firstbot

A lightweight C++ Discord API library supporting the entire Discord API, including Slash Commands, Voice/Audio, Sharding, Clustering and more!

https://dpp.dev/firstbot.html

**Comment by Mastergamer433** [comment]
It does not work. the code can run without bot.on_log(dpp::utility::cout_logger()); but the bot does not come online, and with that line it just have a "segmentation fault (core dumped)". But why does it not work?

**Reply by braindigitalis** [reply]
This is not the correct place for technical support. If you need help with getting your bot to compile or run please visit the discord server.

**Comment by Heco-o** [comment]
It works really nice!
